cmd/soroban-rpc/internal/test/stellar-core-integration-tests.cfg

```diff
@@ -1,5 +1,6 @@
 ARTIFICIALLY_ACCELERATE_TIME_FOR_TESTING=true
 ENABLE_DIAGNOSTICS_FOR_TX_SUBMISSION=true
+DEPRECATED_SQL_LEDGER_STATE=false
```

Comment:
doesn't this require that https://github.com/stellar/go/issues/5295 is implemented?

Reply:
This is for the stellar core node (not captive core)